### PR TITLE
Improve alpine repo automation

### PR DESCRIPTION
**Pull request by osterman**
## what
* Add `Makefile` to orchestrate creation/destruction of cloudformation stack
* Add `terraform/` code to provision CI/CD user for S3 bucket
* Add repo installer script

## why
* Cloud Posse alpine repo for [`cloudposse/packages`](https://github.com/cloudposse/packages)

### CODE REVIEW

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis plan

File: terraform/main.tf [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  backend "s3" {}
+}
+
+variable "aws_assume_role_arn" {
+  type = "string"
+}
+
+variable "domain_name" {
+  type    = "string"
+  default = "cloudposse.com"
```
Comment (aknysh) [inline terraform/main.tf]: cloudposse.co ?

File: terraform/main.tf [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+provider "aws" {
+  assume_role {
+    role_arn = "${var.aws_assume_role_arn}"
+  }
+}
+
+locals {
+  s3_bucket_name = "${var.name}.${var.domain_name}"
```
Comment (aknysh) [inline terraform/main.tf:46]: why do we need `variable "s3_bucket_name"` and local `s3_bucket_name`?

**Review by aknysh — APPROVED** [review]

**Comment by osterman** [comment]
atlantis plan

**Comment by osterman** [comment]
atlantis testing plan

**Comment by osterman** [comment]
atlantis testing plan -p terraform

**Comment by osterman** [comment]
atlantis testing plan -p cloudformation

**Comment by aknysh** [comment]
atlantis testing help

**Comment by aknysh** [comment]
atlantis help

**Comment by osterman** [comment]
atlantis/testing plan

**Comment by osterman** [comment]
atlantis/testing plan

**Comment by aknysh** [comment]
atlantis/testing help

**Comment by aknysh** [comment]
atlantis/testing plan

**Comment by osterman** [comment]
atlantis/testing

**Comment by osterman** [comment]
atlantis/testing plan

**Pull request merged**